src/Dom/scrollLocker.ts

```diff
@@ -17,6 +18,32 @@ const scrollingEffectClassNameReg = new RegExp(
   'g',
 );
 
+let scrollPosition = 0;
+const isIosDevice = /iP(ad|hone|od)|MacIntel/.test(window.navigator.platform);
```

Comment:
We need support ssr, so 

```suggestion
const isIosDevice = canUseDom() && window.navigator &&
  window.navigator.platform &&
  (/iP(ad|hone|od)/.test(window.navigator.platform) ||
    (window.navigator.platform === 'MacIntel' && window.navigator.maxTouchPoints > 1))`;
```

Reply:
Relating document https://stackoverflow.com/questions/41594997/ios-10-safari-prevent-scrolling-behind-a-fixed-overlay-and-maintain-scroll-posi